odt-opt-in/LETTERSPECS/BANNO.ODTOPTIN.V1.CFG

```diff
@@ -122,12 +124,19 @@ AFF:7
 *
 * OVERDRAFT TOLERANCE AMOUNT
 *  Line must begin with "OTA"
-*  Overdraft Tolerance amount by share type.  When opting-in, this amount will be set in the Overdraw Tolerance field
-*  in the share record.  The amount is mapped based on the SHARE:TYPE field.
+*  Overdraft Tolerance amount by share type. When opting-in, this amount will be set in the Overdraw Tolerance field
```

Comment:
I suggest stating in the comments that a maximum of 99 OTA definitions are allowed

Reply:
@jordan-keenan Done